packages/atomic/src/components/search/facets/atomic-category-facet/atomic-category-facet.tsx

```diff
@@ -74,18 +74,18 @@ import {initializePopover} from '../atomic-popover/popover-type';
  * @part search-result-path - The search result path.
  * @part search-highlight - The highlighted query inside the facet values.
  *
- * @part parents - The parent values container.
- * @part all-categories-button - The "View all" button displayed first along the parents.
- * @part parent-button - The clickable parent button.
- * @part active-parent - The non-clickable active parent.
+ * @part parents - The container surrounding the whole hierarchy of values.
+ * @part sub-parents - The container surrounding a part of the hierarchy of values.
+ * @part values - The container surrounding either the children of the active value or the values at the base.
+ * @part all-categories-button - The "View all" button displayed first within the parents.
+ * @part parent-button - The clickable parent button displayed first within sub-parents.
+ * @part active-parent - The clickable active parent displayed first within the last sub-parents.
+ * @part value-link - The clickable value displayed first within values.
  * @part back-arrow - The back arrow displayed before the clickable parents.
- *
- * @part values - The facet values child container.
- * @part value-link - The child facet value.
- * @part value-label - The facet value label.
- * @part value-count - The facet value count.
- * @part leaf-value - A facet value with no child value
- * @part node-value - A facet value with children values
+ * @part value-label - The facet value label within a value button.
+ * @part value-count - The facet value count within a value button.
+ * @part leaf-value - A facet value with no child value.
+ * @part node-value - A facet value with children values.
```

Comment:
```suggestion
 * @part node-value - A facet value with child values.
```
(if you allow me to be a grammar nerd here: 'child' is an adjective for 'values' in this case, so it shouldn't be conjugated. It's easy to miss, especially for french speakers, so it must've slipped through previously.)